tests-execution-chart-commons/src/main/resources/template.html

```diff
@@ -53,24 +53,28 @@
             }
         }
     </style>
-    <script src="@MERMAID_SRC@"></script>
+    <script type="module">
+        import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@10.5.0/dist/mermaid.esm.min.mjs';
```

Comment:
Can we somehow use `@MERMAID_SRC@` instead of this specific url? We would like to give an option to configure url/version. 

There is a configuration option `formats.html.script.embed`, which downloads script. We should still support it.

Reply:
You're right, I was editing compiled html and forgot to change it back to src reference. Thanks for noticing - updated.